doc/source/ray-core/patterns/ray-get-ray-wait-async.rst

```diff
@@ -0,0 +1,41 @@
+Anti-pattern: Calling ray.get and ray.wait in coroutines blocks the event loop
+==============================================================================
+
+**TLDR:** Avoid calling :func:`ray.get() <ray.get>` and :func:`ray.wait() <ray.wait>` in coroutines since they will block the event loop;
```

Comment:
Instead of documenting it, could we raise a warning directly in ray.get()/wait() when using it within a coroutine, and tell the user to use await instead?

Reply:
We already did that. I think it's also good to document it with more details. Probably we can link to it in the warning message.

Reply:
How about we document this inline as a warning directive or something like that in the existing async actors page? Probably it will be more discoverable there than in a separate patterns page.